src/hotspot/cpu/ppc/stubGenerator_ppc.cpp

```diff
@@ -4769,32 +4769,25 @@ address generate_lookup_secondary_supers_table_stub(u1 super_klass_index) {
   }
 
   // load Method* target of MethodHandle
-  // c_rarg0 = jobject receiver
-  // c_rarg1 = JavaThread* thread
+  // R3_ARG1 = jobject receiver
+  // R19_method = result Method*
   address generate_upcall_stub_load_target() {
-    Register rmethod = c_rarg0;
-    Register rreceiver = c_rarg0;
-    Register rthread = c_rarg1;
 
     StubCodeMark mark(this, "StubRoutines", "upcall_stub_load_target");
     address start = __ pc();
     __ save_LR_CR(R0);
```

Comment:
I think `save_LR_CR` and `restore_LR_CR` should get removed, too. CR is not live and LR is preserved everywhere below. But, I'll check this later.